hepcrawl/spiders/cds_spider.py

```diff
@@ -57,23 +49,13 @@ def __init__(self,
 
     def parse_record(self, selector):
         selector.remove_namespaces()
-        try:
-            cds_bibrec, ok, errs = create_bibrec(selector.xpath('.//record').extract()[0])
-            if not ok:
-                raise RuntimeError("Cannot parse record %s: %s", selector, errs)
-            self.logger.info("Here's the record: %s" % cds_bibrec)
-            inspire_bibrec = CDS2Inspire(cds_bibrec).get_record()
-            marcxml_record = record_xml_output(inspire_bibrec)
-            record = create_record(marcxml_record)
-            app = Flask('hepcrawl')
-            app.config.update(
-                self.settings.getdict('MARC_TO_HEP_SETTINGS', {})
-            )
-            with app.app_context():
-                json_record = hep.do(record)
-                base_uri = self.settings['SCHEMA_BASE_URI']
-                json_record['$schema'] = base_uri + 'hep.json'
-            return ParsedItem(record=json_record, record_format='hep')
-        except Exception:
-            logger.exception("Error when parsing record")
-            return None
+        record = create_record(selector.xpath('.//record').extract()[0])
```

Comment:
`extract()[0]` is equivalent to `extract_first()` (the former raises if the list is empty, the latter returns `None` but can be overridden).